src/Ball.java

```diff
@@ -21,6 +21,7 @@ class Ball {
     private int fillColor;
     /** The color of the outside of the ball */
     private int borderColor;
+    public boolean frozen = false;
```

Comment:
using a private variable with a `freeze()` and `unFreeze()` method would be more traditionally java